crates/services/p2p/src/service.rs

```diff
@@ -371,6 +373,11 @@ pub trait Broadcast: Send {
 
     fn tx_broadcast(&self, transaction: TransactionGossipData) -> anyhow::Result<()>;
 
+    fn confirmations_broadcast(
```

Comment:
nit: maybe call it pre-confirmation?

Reply:
Thanks. I iterated on the name and this got left behind.